jobs/ci-job-validation.yaml

```diff
@@ -2,7 +2,6 @@
 - job:
     name: ci-job-validation
     project-type: pipeline
-    sandbox: true
```

Comment:
Will any new scripts now require an extra admin approval after getting merged?

Reply:
no, that is not changing